src/services/ContributingDeveloperAuditService/scm/GitHub/GitHubAudit.ts

```diff
@@ -0,0 +1,38 @@
+import { soosLogger } from "../../../../logging";
+import { IContributingDeveloperAudit } from "../../ContributingDeveloperAuditService";
+import GitHubService from "./GitHubService";
+import { ContributingDeveloper } from "./api/GitHubApiClient";
+import { mergeContributors } from "./utilities";
+
+class GitHubAudit implements IContributingDeveloperAudit {
+  public async audit(
+    implementationParams: Record<string, string>,
+  ): Promise<ContributingDeveloper[]> {
+    const githubPAT = implementationParams["githubPAT"];
```

Comment:
add a validateParams method to the provider interface and call that in the parent service